ci/infra/testrunner/testrunner.py

```diff
@@ -39,15 +39,18 @@ def build_skuba(options):
 
 def bootstrap(options):
     skuba = Skuba(options.conf, options.platform)
-
-    skuba.cluster_init()
+    skuba.cluster_init(kubernetes_version=options.kubernetes_version)
     skuba.node_bootstrap()
 
 
 def cluster_status(options):
     print(Skuba(options.conf, options.platform).cluster_status())
 
 
+def cluster_upgrade(options):
```

Comment:
`cluster_upgrade_plan`

Reply:
i'd still call this `cluster_upgrade_plan` as it doesn't really do a cluster_upgrade